CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Not released yet
 
+# 0.12.0
```

Comment:
That's would be probably the next version, but this is still `Not released yet`. Please remove that version.

Reply:
corrected